crates/sui-graphql-rpc/src/types/event.rs

```diff
@@ -72,16 +76,40 @@ impl Event {
     }
 
     /// Addresses of the senders of the event
-    async fn senders(&self) -> Result<Option<Vec<Address>>> {
-        let mut addrs = Vec::with_capacity(self.stored.senders.len());
-        for sender in &self.stored.senders {
-            let Some(sender) = &sender else { continue };
+    async fn senders(
```

Comment:
Note that in the case of `TransactionBlock` we changed this API to return an `Option<Address>` instead of a list of `senders`:

https://github.com/MystenLabs/sui/blob/9ba167b0af84365188877a448a733a5d877e4ba5/crates/sui-graphql-rpc/src/types/transaction_block.rs#L79-L86

...so I'm thinking that that may be the right thing to do here as well, until we support multiple senders at the protocol level.

Reply:
ah gotcha. Do we need similar logic here checking not SuiAddress::ZERO?

Reply:
I think so -- so that we are consistent.